dbt/adapters/firebolt/connections.py

```diff
@@ -100,9 +100,9 @@ def open(cls, connection):
                     engine_name = credentials.engine_name
                     error_msg_append = ''
                 raise EngineOfflineException(
```

Comment:
There is no need for this exception since SDK checks running engine by itself and raises an appropriate error

Reply:
kk